rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -0,0 +1,80 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_maintain
+%global confdir foreman_maintain
+
+%{!?_root_bindir:%global _root_bindir %{_bindir}}
+
+Summary: The Foreman/Satellite maintenance tool
```

Comment:
The motivation for the tool is the very same version to be useful both for upstream (Foreman) as well as downstream (Satellite or any other downstream that will use Foreman as upstream). There is no plan to do a separate, downstream only version of it and I would like to encourage sharing here, rather than forking. That's why it's different from other projects in the organization, and why other projects don't mention Satellite.